assets/translations/es/messages.private.json

```diff
@@ -0,0 +1,64 @@
+{
```

Comment:
Also just a note: when we get to providing translations for our current beta helplines (in Italian, Greek, Khmer, etc), I don't think all of these strings will need to be translated.  Just the strings that get exposed in the current chat app, which is only maybe 3 or 4 I think.

Reply:
Yes, I though about this and dropped them in the same file, because this refers to what a caller will see (no matter the chat channel of choice). Should I split them in two separate files? Maybe translate the webchat strings within the code? Right now they don't share any string (webchat have html tags inside GoodByeMsg).

Reply:
Yeah, given that there's basically no overlap, I don't think they need to pull from the same file as the UI.  I'm okay with putting them right in the code here or in a separate file for now.  If we put them in the code they should be in the same kind of JSON format we would use in a file.